staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/types_jsonschema.go

```diff
@@ -109,6 +109,18 @@ type JSONSchemaProps struct {
 	//      used to identify them. Order is preserved upon merge. The map tag
 	//      must only be used on a list with elements of type object.
 	XListType *string
+
+	// x-kubernetes-map-type annotates a map to further describe its topology.
```

Comment:
We're actually using this on both maps and structs, we could probably change to:

> x-kubernetes-map-type annotates an object (or maybe type?) to further describe ...